testing/tests/common/mod.rs

```diff
@@ -358,41 +359,16 @@ impl ApplicationMetadata for StaticCommitteeMetaData {}
 )]
 pub struct StaticCommitteeTestTypes;
 impl NodeType for StaticCommitteeTestTypes {
-    // TODO (da) can this be SequencingConsensus?
     type ConsensusType = ValidatingConsensus;
     type Time = ViewNumber;
-    type BlockType = DummyBlock;
```

Comment:
It looks like they're used in the memory_storage tests and for the vrf integration tests (which I don't think matter since we're not using vrf with sortition anymore). I'm okay leaving it in as a "simpler than the demo" thing used for unit tests only.

Reply:
Yeah sounds good to me!